### DISCUSSION TITLE
will be there any support for secrets?

### DISCUSSION THREAD

**Discussion by marlon-rodriguezr** [databrickslabs/dbx#574] [Q&A]
running a test with Jinja enabled (to differentiate multiple environments), I got an error message because there was a secret var resolution inside the deployment.yml so, is there any plan to handle secrets/env variables the usual way we do in Databricks workspace or the integration of these variables with the Jinja ones?

**Comment by renardeinside** [comment]
You simply need to escape your secrets string, e.g. instead of:

```
spark_env_vars:
  MY_SECRET: {{secrets/someScope/someName}}
```

use this in case if Jinja templating is enabled:

```
spark_env_vars:
  MY_SECRET: {{ "{{secrets/someScope/someName}}" }}
```

This will prevent Jinja from trying to resolve the string and properly pass it to the Jobs API.
